modules/rosa-create-objects.adoc

```diff
@@ -109,78 +107,99 @@ Create a new cluster.
 
-|--host-prefix
-|The subnet prefix length (integer) to assign to each individual node. For example, if host prefix is set to `23`, then each node is assigned a `/23` subnet out of the given CIDR.
+|--host-prefix <subnet>
+|The subnet prefix length to assign to each individual node, as an integer. For example, if host prefix is set to `23`, then each node is assigned a `/23` subnet out of the given CIDR.
 
-|--machine-cidr
-|Block of IP addresses (ipNet) used by {product-title} while installing the cluster. Example: `10.0.0.0/16`
+|--machine-cidr <address_block>
+a|Block of IP addresses (ipNet) used by {product-title} while installing the cluster, for example, `10.0.0.0/16`.
 
-|--max-replicas
+[IMPORTANT]
+====
+OVN-Kubernetes, the default network provider in {product-title} 4.11 and later, uses the `100.64.0.0/16` IP address range internally. If your cluster uses OVN-Kubernetes, do not include the `100.64.0.0/16` IP address range in any other CIDR definitions in your cluster.
+====
+
+|--max-replicas <number_of_nodes>
 |Specifies the maximum number of compute nodes when enabling autoscaling. Default: `2`
 
-|--min-replicas
+|--min-replicas <number_of_nodes>
 |Specifies the minimum number of compute nodes when enabling autoscaling. Default: `2`
 
 |--multi-az
 |Deploys to multiple data centers.
 
-|--pod-cidr
-|Block of IP addresses (ipNet) from which pod IP addresses are allocated. Example: `10.128.0.0/14`
+|--operator-roles-prefix <string>
+|Prefix to use for all IAM roles used by the operators needed in the OpenShift installer. A prefix is generated automatically if you do not specify one.
+
+|--pod-cidr <address_block>
+a|Block of IP addresses (ipNet) from which pod IP addresses are allocated, for example, `10.128.0.0/14`.
+
+[IMPORTANT]
+====
+OVN-Kubernetes, the default network provider in {product-title} 4.11 and later, uses the `100.64.0.0/16` IP address range internally. If your cluster uses OVN-Kubernetes, do not include the `100.64.0.0/16` IP address range in any other CIDR definitions in your cluster.
+====
 
 |--private
 |Restricts primary API endpoint and application routes to direct, private connectivity.
 
 |--private-link
-| Specifies to use AWS PrivateLink to provide private connectivity between VPCs and services. The `--subnet-ids` argument is required when using `--private-link`.
+|Specifies to use AWS PrivateLink to provide private connectivity between VPCs and services. The `--subnet-ids` argument is required when using `--private-link`.
+
+|--region <region_name>
+|The name of the AWS region where your worker pool will be located, for example, `us-east-1`. This argument overrides the `AWS_REGION` environment variable.
+
+|--role-arn <arn>
+|The Amazon Resource Name (ARN) of the installer role that {cluster-manager} uses to create the cluster. This is required if you have not already created account roles.
```

Comment:
Can you just use ARN since you've spelled out previously in this module?

Reply:
We could, but since this parameter relates to ARNs specifically I think it's more useful to spell it out in full here.